### ISSUE TITLE
data.pickle corrupted after setting trailing SL

### ISSUE THREAD

**Issue by offtopic** [MarcelBeining/EazeBot#17]
Hi, I again :)

I wanted to set a trailing SL  in percent:

Error:
```
2020-08-11 13:34:48,017 [Bot:73674776] [INFO ]  Finished updating trade sets...
Traceback (most recent call last):
  File "C:\Users\thoma\AppData\Local\Programs\Python\Python37-32\lib\site-packages\eazebot\bot.py", line 322, in print_trade_status
    'showProfitIn']),
  File "C:\Users\thoma\AppData\Local\Programs\Python\Python37-32\lib\site-packages\eazebot\tradeHandler.py", line 415, in get_trade_set_info
    ' (trailing with offset %.2g %%)' % (ts.delta * 100)))
AttributeError: 'BaseTradeSet' object has no attribute 'delta'
2020-08-11 13:34:51,879 [Bot:73674776] [ERROR]  None
2020-08-11 13:35:04,372 [Bot:73674776] [INFO ]  Updating trade sets...
2020-08-11 13:35:18,042 [Bot:73674776] [INFO ]  Finished updating trade sets...
Traceback (most recent call last):
  File "C:\Users\thoma\AppData\Local\Programs\Python\Python37-32\lib\site-packages\eazebot\bot.py", line 322, in print_trade_status
    'showProfitIn']),
  File "C:\Users\thoma\AppData\Local\Programs\Python\Python37-32\lib\site-packages\eazebot\tradeHandler.py", line 415, in get_trade_set_info
    ' (trailing with offset %.2g %%)' % (ts.delta * 100)))
AttributeError: 'BaseTradeSet' object has no attribute 'delta'
2020-08-11 13:35:20,690 [Bot:73674776] [ERROR]  None

```
After the error the trade set has completely disappeared, i tried to restart the bot, but now the data file is completly corrupted:

```
******** Welcome to EazeBot (v2.6.1) ********
Free python/telegram bot for easy execution & surveillance of crypto trading plans on multiple exchanges


C:\Users\thoma\AppData\Local\Programs\Python\Python37-32\lib\site-packages\telegram\ext\conversationhandler.py:220: UserWarning: If 'per_message=False', 'CallbackQueryHandler' will not be tracked for every message.
  warnings.warn("If 'per_message=False', 'CallbackQueryHandler' will not be "
2020-08-11 13:40:56,068 [MainThread  ] [INFO ]  Loading user data
Traceback (most recent call last):
  File "C:\Users\thoma\AppData\Local\Programs\Python\Python37-32\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "C:\Users\thoma\AppData\Local\Programs\Python\Python37-32\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\thoma\AppData\Local\Programs\Python\Python37-32\lib\site-packages\eazebot\__main__.py", line 12, in <module>
    main.main()
  File "C:\Users\thoma\AppData\Local\Programs\Python\Python37-32\lib\site-packages\eazebot\main.py", line 106, in main
    EazeBot(config=config).start_bot()
  File "C:\Users\thoma\AppData\Local\Programs\Python\Python37-32\lib\site-packages\eazebot\bot.py", line 1464, in start_bot
    self.updater.dispatcher.user_data = clean_data(load_data(), self.__config__['telegramUserId'])
  File "C:\Users\thoma\AppData\Local\Programs\Python\Python37-32\lib\site-packages\eazebot\auxiliary_methods.py", line 307, in load_data
    raise e
  File "C:\Users\thoma\AppData\Local\Programs\Python\Python37-32\lib\site-packages\eazebot\auxiliary_methods.py", line 287, in load_data
    return dill.load(f)
  File "C:\Users\thoma\AppData\Local\Programs\Python\Python37-32\lib\site-packages\dill\_dill.py", line 305, in load
    obj = pik.load()
EOFError: Ran out of input
```

Standard version: 2.6.1
OS: Windows 10 64 bit
ccxt: 1.32.12

Any help would be very appreciate ...
I tried the backup, but there exists only one backup(?) which is already corrupted :(

```
{
	"telegramAPI": "xxxxxxxxxx",
	"telegramUserId": xxxxxxxx,
	"updateInterval": 0.5,
	"extraBackupInterval": 7,
	"maxBackupFileCount": 12,
	"minBalanceInBTC": 0.001
}
```

Thx and best regards,
off

**Comment by offtopic** [comment]
I think the first issues is related to the missing ts.**SL**.delta

tradeHandler.py -> #415

\- ``` ' (trailing with offset %.2g %%)' % (ts.delta * 100))) ```
\+ ``` ' (trailing with offset %.2g %%)' % (ts.SL.delta * 100))) ```

**Comment by MarcelBeining** [comment]
Hi, sorry, my mail program did not inform me about new mails since a week :-D I will cope with your issues tomorrow. Eazebot has also the function to backup additional pickle files in a folder "backup" each x days. Have you activated this feature (see config template in git folder eazebot/templates)

**Comment by MarcelBeining** [comment]
Fixed in [v2.6.2](https://github.com/MarcelBeining/EazeBot/releases/tag/v2.6.2)

Thanks for your patience

**Comment by offtopic** [comment]
Regarding backup -> i have backups for the non-docker version, but i don't have backups for the docker version
(also no log telegramEazeBot.log.* files) -> same config
